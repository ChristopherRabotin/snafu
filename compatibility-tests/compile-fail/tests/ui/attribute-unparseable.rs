use snafu::prelude::*;

#[derive(Debug, Snafu)]
enum InnerError {
    InnerVariant,
}

#[derive(Debug, Snafu)]
enum Error {
    AVariant {
        // Invalid identifier, and not a boolean; should error
        #[snafu(source(5))]
        a: InnerError,
    },
}

fn main() {}
