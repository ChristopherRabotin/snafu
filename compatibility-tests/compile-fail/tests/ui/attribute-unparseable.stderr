error: expected boolean literal or `from`
  --> tests/ui/attribute-unparseable.rs:12:24
   |
12 |         #[snafu(source(5))]
   |                        ^
